mindsdb/interfaces/knowledge_base/controller.py

```diff
@@ -1159,6 +1159,12 @@ def list(self, project_name: str = None) -> List[dict]:
 
         return data
 
+    def create_index(self, table_name, project_name):
+        project_id = self.session.database_controller.get_project(project_name).id
+        kb_table = self.get_table(table_name, project_id)
+        db_handler = kb_table.get_vector_db()
+        db_handler.create_index(table_name=kb_table._kb.vector_database_table)
```

Comment:
let's create method in KnowledgeBaseTable to get table name (to not use private variable)

or maybe better:

we can move `create index` logic into KnowledgeBaseTable class and left here only:
- kb_table = self.get_table(table_name, project_id)
-  kb_table.create_index()